### PR TITLE
Bigtable: add enhanced stub for bigtable table admin client

### PR DESCRIPTION
**Pull request by igorbernstein2**
This will be used #3658 to add new callables that can't be autogenerated.

### CODE REVIEW

**Comment by igorbernstein2** [comment]
@kevinsi4508 please take a look

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.admin.v2.stub;
+
+import com.google.api.core.InternalApi;
+import com.google.api.gax.retrying.RetrySettings;
```
[the remark this answers is not in the document]
Comment (kevinsi4508) [reply google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java]: Remove RetrySettings

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import com.google.api.gax.rpc.ClientContext;
+import com.google.api.gax.rpc.UnaryCallable;
```
[the remark this answers is not in the document]
Comment (kevinsi4508) [reply google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java]: Remove UnaryCallable

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import com.google.bigtable.admin.v2.TableName;
+import java.io.IOException;
+import org.threeten.bp.Duration;
```
[the remark this answers is not in the document]
Comment (kevinsi4508) [reply google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java]: Remove Duration

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.admin.v2.stub;
+
+import com.google.api.core.InternalApi;
+import com.google.api.gax.retrying.RetrySettings;
+import com.google.api.gax.rpc.ClientContext;
+import com.google.api.gax.rpc.UnaryCallable;
+import com.google.bigtable.admin.v2.TableName;
```
[the remark this answers is not in the document]
Comment (kevinsi4508) [reply google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java]: Remove TableName

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+ */
+@InternalApi
+public class EnhancedBigtableTableAdminStub extends GrpcBigtableTableAdminStub {
+  private final BigtableTableAdminStubSettings settings;
+  private final ClientContext clientContext;
+
+  public static EnhancedBigtableTableAdminStub createEnhanced(
+      BigtableTableAdminStubSettings settings)
+      throws IOException {
+
```
Comment (kevinsi4508) [inline google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java]: Remove empty line?

**Comment by igorbernstein2** [comment]
Removed all unused imports & whitespace

**Comment by igorbernstein2** [comment]
@pongad can you take a look when you have a moment?

**Pull request merged**